docs/docs/concepts/jobs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Jobs
+sidebar_position: 6
+---
+
+Jobs can be handled with [dm-job](https://github.com/equinor/dm-job).
+
+This is an API where you can:
+- Start jobs
+- Stop jobs
+- Request the status for a running job
+- Get result from a job
+
+
+If your custom application want to use dm-job, you have to create a JobHandler file for your job.
+
+This JobHandler must have the name `__init__.py` and be placed inside a folder called `job_handler_plugins`.
+
+To run a job, you must define a job entity. Example:
+```json
+{
+    "label": "Example local container job",
+    "type": "dmss://WorkflowDS/Blueprints/Job",
+    "status": "Not started",
+    "triggeredBy": "author",
+    "outputTarget": "dmss://DataSource/$456-456.result"
+    "applicationInput": {
+      "type": "dmss://system/SIMOS/Reference",
+      "referenceType": "link",
+      "address": "dmss://DataSource/$123-123",
+    },
+    "runner": {
+      "type": "dmss://DataSource/apps/MySignalApp/models/SignalGeneratorJob",
+    },
+    "started": "Not started",
+  }
+```
+
+
+The JobHandler file (`__init__.py`) has the following structure:
+
+```python
+_SUPPORTED_TYPE = "dmss://WorkflowDS/Blueprints/Container"
+
+
+class JobHandler(JobHandlerInterface):
+    def __init__(self, job: Job, data_source: str):
+        super().__init__(job, data_source)
+        self.headers = {"Access-Key": job.token}
+
+    def start(self) -> str:
+        # Code for running the job
+
+    def remove(self) -> Tuple[str, str]:
+        # Remove a job
+
+    def progress(self) -> Tuple[JobStatus, str]:
+        """Poll progress from the job instance"""
+        
+
+    def result(self) -> Tuple[str, bytes]:
+        # Code for returning the response from a job.
+
+```
+
+A few important things to note:
+- A defined variable called `_SUPPORTED_TYPE` decides what JobHandler to use for a given job. This is the same as the runner type in the job entity.
+- The JobHandler and Job class are defined in dm-job repository.
+- To access the job entity, use `self.job.entity`
+- - `self.job.entity.applicationInput` is used for supplying inputs to the job. For example, you can include inputs to calculations.
+- - `self.job.entity.outputTarget` is used for defining where the result of the job should be stored.
+
+
+
+Jobs can be run in different ways:
+- locally in the instance / deployment of the dm-job API, either as a "container job" or a "regular job". For container jobs, the job enitty's runner type should extend from the  `Container` blueprint from dm-job. For regular jobs, the job entity's runner type should extend from the `JobHandler` blueprint from dm-job.
+- Azure container jobs. For this case, the job entity's runner type should extend from `AzureContainer´blueprint in dm-job. 
```

Comment:
When should I choose one over the other?

Reply:
The user can in theory choose either a local job or container for all jobs. However, it's better to use Azure jobs for jobs that take a long time, since they can be run in the background. Will add this to the docs.